appinfo/info.xml

```diff
@@ -5,14 +5,15 @@
 	<description>Collabora Online allows you to to work with all kinds of office documents directly in your browser. This application can connect to a Collabora Online server (WOPI Client). ownCloud is the WOPI Host. Please read the documentation to learn more about that.</description>
 	<summary>Edit office documents directly in your browser.</summary>
 	<licence>AGPL</licence>
-	<version>2.0.9</version>
+	<version>2.0.10</version>
 	<author>Collabora Productivity based on work of Frank Karlitschek, Victor Dubiniuk</author>
 	<bugs>https://github.com/owncloud/richdocuments/issues</bugs>
 	<repository type="git">https://github.com/owncloud/richdocuments.git</repository>
 	<category>office</category>
 	<category>integration</category>
 	<types>
 		<prevent_group_restriction/>
+		<filesystem/>
```

Comment:
@PVince81  do I really need it? What this tag gives me?

Reply:
without this none of the file event hooks will be triggered.
it is also possible that the FS will not be setup with 'setupFS()`